Documentation/CodingGuidelines

```diff
@@ -566,8 +565,8 @@ Writing Documentation:
      --short:: Use this to emit output in the short-format.
      --short:: You can use this to get output in the short-format.
      --short:: A user who prefers shorter output could....
-     --short:: Should a person and/or program want shorter output, he
-               she/they/it can...
+     --short:: Should a person and/or program want shorter output,
+               he/she/they can...
```

Comment:
This part of the change is good, even if the other parts may end up needing discarded.